readme-vars.yml

```diff
@@ -0,0 +1,91 @@
+---
+
+# project information
+project_name: rustdesk
+project_url: "https://rustdesk.com/"
+project_logo: "https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/rustdesk-logo.png"
+project_blurb: "[RustDesk]({{ project_url }}) is a full-featured open source remote control alternative for self-hosting and security with minimal configuration."
+project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+project_categories: "Web"
+project_blurb_optional_extras_enabled: false
+
+# supported architectures
+available_architectures:
+  - { arch: "{{ arch_x86_64 }}", tag: "latest"}
+  - { arch: "{{ arch_arm64 }}", tag: "arm64v8-latest"}
+
+# development version
+development_versions: false
+
+# container parameters
+common_param_env_vars_enabled: true
+param_container_name: "{{ project_name }}"
+param_usage_include_env: true
+param_env_vars:
+  - { env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use EG Europe/London." }
```

Comment:
TZ is not read by jenkins anymore, it is implied as UTC  when common_param_env_vars_enabled is true